doc/plutus/explanations/plutus-tools-component-descriptions.rst

```diff
@@ -0,0 +1,321 @@
+The PAB executes the off-chain component of Plutus applications. It manages 
+application requests to the wallet backend, the Cardano node and the chain-index. 
+PAB stores the application state and offers an HTTP REST API for managing application 
+instances. 
+
+PAB wraps the contracts built with plutus-contract. It is the central point of 
+contact for everything, integrating all the Cardano components. 
+
++--------------+---------------------------------------------------------------+
+| PAB                                                                          |
++==============+===============================================================+
+| Location     | ``plutus-apps/plutus-pab``                                    |
++--------------+---------------------------------------------------------------+
+| Dependencies | cardano-api, plutus-ledger-api, plutus-ledger,                |
+|              | plutus-contract, plutus-contract emulator,                    |
+|              | plutus-chain-index-core                                       |
++--------------+---------------------------------------------------------------+
+| State        | Maintenance mode for bugs, but not a high priority component. |
+|              | We are not adding any new features unless they are            |
+|              | contributed by external stakeholders.                         | 
++--------------+---------------------------------------------------------------+
+
+13. PAB simulator 
+---------------------------
+
+`PAB simulator <https://github.com/input-output-hk/plutus-apps/tree/main/plutus-pab>`_ 
+is a simulator for the PAB that is used for testing a plutus application in a 
+simulated environment. 
+
++--------------+---------------------------------------------------------------+
+| PAB simulator                                                                |
++==============+===============================================================+
+| Location     | ``plutus-apps/plutus-pab``                                    |
++--------------+---------------------------------------------------------------+
+| Dependencies | cardano-api, plutus-ledger-api, plutus-ledger,                |
+|              | plutus-contract, plutus-contract emulator,                    |
+|              | plutus-chain-index-core, plutus-pab                           |
++--------------+---------------------------------------------------------------+
+| State        | Maintenance mode for bugs. Not a high priority component.     | 
++--------------+---------------------------------------------------------------+
+
+14. Plutus application template (aka Plutus platform starter project)
+---------------------------------------------------------------------------
+
+`Plutus application template <https://github.com/input-output-hk/plutus-starter>`_ 
+is a Haskell template offering a simple starter project to assist dApp 
+developers in starting to write Plutus applications. 
+
```

Comment:
Is this one deprecated?

Reply:
In it's current form, yes. But it's a must-have in the future because it's a way of getting entry-level plutus-application developers into the field. Do we want to remove it in the time being?

Reply:
I think people keep trying to use it still so it might be worth deprecating for now

Reply:
Until we have a version of this that we can genuinely promote, seems like it would be best not to mention it. I'll remove it for this version of the tools overview page.